app/controllers/api/users.php

```diff
@@ -322,8 +318,8 @@
             $record = $geodb->get($log['ip']);
 
             if ($record) {
-                $output[$i]['countryCode'] = (isset($countries[$record['country']['iso_code']])) ? \strtolower($record['country']['iso_code']) : '--';
-                $output[$i]['countryName'] = (isset($countries[$record['country']['iso_code']])) ? $countries[$record['country']['iso_code']] : $locale->getText('locale.country.unknown');
+                $output[$i]['countryCode'] = $locale->getText('countries.'.strtolower($record['country']['iso_code']), '--');
```

Comment:
Again, this portion of code just needs the country code.

Reply:
![Screenshot from 2021-07-26 10-47-03](https://user-images.githubusercontent.com/20852629/126936781-f1b816ab-030a-4808-96ac-a635c702a14f.png)


I dont think so... 
`$countries = $locale->getText('countries'); `

which returns 
```php
[
   "IN" => "India"
]
```

Reply:
Done :+1: